torch/distributed/_composable/fsdp/_fsdp_init.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from torch._prims_common import DeviceLikeType
+
+
+def _normalize_device(device: DeviceLikeType) -> torch.device:
+    if isinstance(device, torch.device):
+        if device == torch.device("cuda"):
+            return torch.device("cuda", torch.cuda.current_device())
+        return device
+    elif isinstance(device, int):
+        return torch.device("cuda", device)
```

Comment:
it looks generic enough to be a pytorch-level utils except this line. We seems to assume ``int/1`` referring to ``cuda:1``?

Reply:
I think that assumption is already made in `torch`:
```
>>> import torch
torch>>> torch.device(1)
device(type='cuda', index=1)
```